# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchTypes.ipdlh
# Commit: 6cbb442f4772
# Full Hash: 6cbb442f477260469cab9f1d81e33fcaede82029
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-18 09:51:31
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r dbdca4661a35 -r 6cbb442f4772 dom/fetch/FetchTypes.ipdlh
--- a/dom/fetch/FetchTypes.ipdlh	Tue Jan 17 15:13:45 2023 +0000
+++ b/dom/fetch/FetchTypes.ipdlh	Tue Jan 17 15:13:46 2023 +0000
@@ -35,8 +35,9 @@
   nsID uuid;
 };
 
-struct ParentToChildStream {
-  RemoteLazyInputStream stream;
+union ParentToChildStream {
+  RemoteLazyInputStream;
+  IPCStream;
 };
 
 struct ChildToParentStream {
@@ -86,6 +87,9 @@
   nsCString alternativeDataType;
   nsITransportSecurityInfo securityInfo;
   PrincipalInfo? principalInfo;
+  nsCString bodyBlobURISpec;
+  nsString bodyLocalPath;
+  RequestCredentials credentialsMode;
 };
 
 struct ParentToParentInternalResponse {